website/src/css/imports/docs.css

```diff
@@ -163,23 +169,10 @@
 
 	h3 {
 		color: $grey;
-		font-size: 12px;
-		font-weight: $semibold;
-		text-transform: uppercase;
-		letter-spacing: 1.5px;
-		margin-top: $medium;
+		font-size: 20px;
+		margin-top: $large;
```

Comment:
Switched from very small uppercase header with a decorative divider to a standard header that more clearly marks the section.